src/Input/Input.jsx

```diff
@@ -131,8 +133,10 @@ export const Input = forwardRef(
             type={inputType}
             value={value}
             variant={inputVariant}
+            onClick={onInputClick}
```

Comment:
do we need to have this explicit in here ? how about using directly the onClick from ...rest instead of onInputClick ...

Reply:
We use the props from `rest` in the input wrapper (label+ input field+subLabel) and in the same time in the input field as well.. so I would leave it explicit for now